### ISSUE TITLE
Peer to peer code review

### ISSUE THREAD

**Issue by Dangelo-JAN** [fed1k/Personal-Portfolio.github.io#19]
These are some suggestions for your project:

- The <img id="dot"> should be styled in CSS instead of with the img element, because the alt text is irrelevant to the content of the game. styling with CSS will improve the experience of a blind reader or anyone using a screen reader.
- To use patterm property in the inputs for validations
